### ISSUE TITLE
Make sure correct env is invoked

### ISSUE THREAD

**Issue by bric3** [clvv/fasd#57]
Fun fact activemq has an `env` shell script if you happen to prepend `$PATH` with your personnal paths like : 

```sh
export PATH=/opt/activemq/bin:$PATH
```

Any script won't work without failing, because ActiveMQ's `env` returns `0`. In **fasd** this line won't do anything : 

```sh
env mv -f "$tempfile" "$_FASD_DATA"
```

The symptom is that **fasd** won't jump because `--add` or `--delete` commands won't add your paths in the data file (usually `.fasd`). So **fasd** doesn't know these directories.

Another option would be to remove `env`, thoughts ?

**Comment by clvv** [comment]
activemq should rename their `env` shell script. Or they should tell you not to export their directory to `$PATH`. `env` is a standard POSIX command. http://pubs.opengroup.org/onlinepubs/9699919799/

**Comment by bric3** [comment]
Yeah I know, I've made a PR on their project renaming this script.

Thx for the link it will allow this PR to hove a bit more weight. But I wonder if the proper `env`could be enforced.

Any way this issue has been written to _log_ the issue. Maybe that could be useful to someone :)

**Comment by clvv** [comment]
This is much appreciated. I just realized that the link I provided earlier did not link to the actual page for `env`. This is now fixed.
